services:
  people_tracking:
    start_tracking: "/butia_vision/pt/start"
    stop_tracking: "/butia_vision/pt/stop"
  image_server:
    image_request: "/butia_vision/is/image_request"
    
topics:
  object_recognition:
    people_detection: "/butia_vision/or/people_detection"
  people_tracking:
    people_tracking: "/butia_vision/pt/people_tracking"